src/MongoDB.Driver/Encryption/ClientEncryption.cs

```diff
@@ -39,11 +40,17 @@ public sealed class ClientEncryption : IDisposable
         /// <param name="clientEncryptionOptions">The client encryption options.</param>
         public ClientEncryption(ClientEncryptionOptions clientEncryptionOptions)
         {
-            _cryptClient = CryptClientCreator.CreateCryptClient(
+            var cryptClientSettings = new CryptClientSettings(
                 bypassQueryAnalysis: null,
+                csfleLibPath: null,
+                csfleSearchPath: null,
                 encryptedFieldsMap: null,
+                isCsfleRequired: null,
                 kmsProviders: clientEncryptionOptions.KmsProviders,
                 schemaMap: null);
+
+            _cryptClient = CryptClientCreator.CreateCryptClient(cryptClientSettings);
```

Comment:
I think it's better to add a factory method into clientEncryptionOptions like `clientEncryptionOptions.CreateCryptClientSettings()`